src/helpers/onEvent.test.ts

```diff
@@ -0,0 +1,20 @@
+import { onEvent } from './onEvent';
+
+describe('Creating event handlers', () => {
+	// We shouldn't have to worry about testing with poorly-formatted arguments.
+	// Since this is an internal tool, not one for a public SDK, we expect
+	// TypeScript to yell at any dev silly enough to include too many or too
+	// few arguments, or arguments of the wrong type.
+
+	test('creates a proper event handler', () => {
+		const execute = (): void => undefined;
```

Comment:
I'm pretty sure the event handler for `ready` receives a `client` parameter. Did TypeScript not yell at you about that?

Reply:
You are correct that this function would receive a `client` parameter, but because that parameter is never used, and it is the only one, it may be omitted here. This is valid TypeScript.

This lambda technically _would_ receive an argument of type `Client<true>`, which may be found in its `arguments` list at runtime if we were to check that, but listing that param here is optional.